### PR TITLE
Update cli.md with more specifics about deployment

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by johnnyman727** [comment]
looks good to me

**Comment by HipsterBrown** [comment]
Awesome! 👍 LGTM

**Comment by rwaldron** [comment]
@Frijol ?

**Comment by Frijol** [comment]
Is there an issue this is tied to? Looking for the problem statement this resolves.

File: cli.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Along with the package.json and index.js included in the `t2 init` process, ther
 * `t2 rename` change the name of a Tessel
 
 ### Code Deployment 
-During code deployment, CLI looks for `.tesselignore` and `.tesselinclude` files to let it know which files it should bundle up and push over to Tessel. In the default bundling process, CLI takes the file passed into the `run` or `push` commands and finds all its dependencies by following the 'require' statements (we use [Browserify](https://github.com/substack/browserify-handbook#how-browserify-works) to do this).
+During code deployment, CLI looks for `.tesselignore` and `.tesselinclude` files to let it know which files it should bundle up and push over to Tessel. In the default bundling process, CLI takes the file passed into the `run` or `push` command and uses it as an entry point to build a dependency graph (similar to [Browserify](https://github.com/substack/browserify-handbook#how-browserify-works)). Once the dependencies are known, binary modules are replaced by pre-compiled (for mips) binaries, assets are copied and everything this tarred and sent to the awaiting Tessel.
```
Comment (Frijol) [inline cli.md]: typo: "assets are copied and everything this tarred and sent"

**Comment by HipsterBrown** [comment]
@Frijol There is no direct issue, but the content around deployment is inaccurate and this PR should clear up some of those details. I pushed a fix to that typo you pointed out as well. 

LGTM 👍

**Comment by Frijol** [comment]
oh, I see! Sounds good.

**Pull request merged**